dexfile/src/main/kotlin/com/github/netomi/bat/dexfile/instruction/editor/InstructionBuilder.kt

```diff
@@ -53,10 +61,22 @@ class InstructionBuilder private constructor(private val dexEditor: DexEditor) {
         return add(FillArrayDataInstruction.of(payload, register))
     }
 
+    fun arrayGetByte(destinationRegister: Int, arrayIndex: Int, index: Int): ArrayInstruction {
```

Comment:
the second parameter should be named arrayRegister as arrayIndex is a bit confusing.